include/Storage.h

```diff
@@ -5,7 +5,7 @@
 union Storage
 {
     int i;
-    double f;
```

Comment:
Hat das etwas it dem QT-Upgrade zu tun?

Es kann gut sein, dass die von-Neumann-Architektur als 32-bit gedacht ist, aber das würde ich gern separat diskutieren. Eventuell untermauert mit dem Skript. Es ist schwer sich zu erinnern warum wir damals so entschieden habe - es kann sein, dass die damalige Delpi-Version ausschlaggebend war.

Reply:
Der Vorschlag kam von Herrn Jakobi--er meinte, die Fließkommazahlen sollten intern als 32-Bit-Floats repräsentiert werden.